src/Juvix/Compiler/Backend/Geb/Translation/FromSource.hs

```diff
@@ -25,23 +25,23 @@ fromSource ::
   Path Abs File ->
   Text ->
   Sem r Geb.Expression
-fromSource fileName input =
-  case runParser fileName input of
+fromSource fileName input' =
+  case runParser fileName input' of
```

Comment:
These `input` name changes really should be in a separate PR, if possible. It's too many of them,

Reply:
This change was moved to https://github.com/anoma/juvix/pull/2582